src/lib/components/SentryAppLink.tsx

```diff
@@ -18,7 +22,7 @@ export default function SentryAppLink({children, to, onClick}: Props) {
   });
 
   return (
-    <a href={url} onClick={onClick} rel="noreferrer noopener" target="_blank" className="text-blue-400">
+    <a href={url} onClick={onClick} rel="noreferrer noopener" target="_blank" className={className ?? linkClass}>
```

Comment:
I would expect these classes to be merged, otherwise e.g. adding a margin on a link would wipe out it's styling